### ISSUE TITLE
Compile it?

### ISSUE THREAD

**Issue by Resoluciones** [GorgeousOne/TangledMaze#1]
Hi!

Would it be possible you compile it, so I can test it?

Thanks!!!!

**Comment by GorgeousOne** [comment]
@Resoluciones In my last commit i wrote that it is complete :D Or do you need it as a jar file?

**Comment by Resoluciones** [comment]
Yes pls. I mean I jar file.  I tried to compile it myself, but as it doesn't have the pom.xml it is difficult for me as I don't have the knowledge.

Thank you!!!

**Comment by GorgeousOne** [comment]
ok here you go with a test version. its right on the first page

**Comment by Resoluciones** [comment]
sorry for the delay, when I realized that the plugin does not make the mazes automatically, I could not test it.  I wrongly thought the plugin worked without an operator, creating the mazes itself underground in active chunks.

**Comment by GorgeousOne** [comment]
Do you need further instructions or are the help pages enough? (I updated them a few times, the last version update be the best)

**Comment by GorgeousOne** [comment]
Are you still going to try it out or did you lost the interest in it?
